Secp256k1.Native.targets

```diff
@@ -1,32 +1,33 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+      <RuntimeIdentifiers>win-x64;win-x86;osx-x64;linux-x64</RuntimeIdentifiers>
+    </PropertyGroup>    
   <ItemGroup Condition="'$(SkipSecp256k1NativeLibCopy)' != 'true'">
-
-    
     <Content Include="$(MSBuildThisFileDirectory)..\..\runtimes\win-x64\native\secp256k1.dll">
       <PackageCopyToOutput>true</PackageCopyToOutput>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory Condition="'$(RuntimeIdentifier)'=='win-x64'">PreserveNewest</CopyToOutputDirectory>
```

Comment:
Is `RuntimeIdentifier` always set with the various ways projects can be built? If not, seems like this condition should account for that.

Reply:
I see,  thank you for comment,  I'll investigate this additionally and will add a fix or a clarification.